examples/platform/set_user_enable.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Set a user's enable/disable status from the command line."""
+
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import User
+import sys
+
+
+def peek_status(user):
+    """Returns True if user is enabled, False if disabled."""
+    grant = user.grant()
+    return not grant.revoked
+
+
+def poke_status(user, value):
```

Comment:
This should be `get_status` and `update_status` as the function names are not intuitive

Reply:
You never coded in BASIC on the old late-70's micros, did you? In any event, two instances of Refactor -> Rename. :)